dev-tools/build.sh

```diff
@@ -9,7 +9,7 @@ composer require --no-update symfony/polyfill-mbstring
 
 # install package deps without dev-deps / remove already installed dev-deps
 # box can ignore dev-deps, but dev-deps, when installed, may lower version of prod-deps
-composer update --no-interaction --no-progress --no-dev --prefer-stable
+composer update --no-interaction --no-progress --no-dev --prefer-stable --optimize-autoloader
```

Comment:
Let's go even further and do `--classmap-authoritative`.

Reply:
i'm not sure, we were not using it before + https://getcomposer.org/doc/articles/autoloader-optimization.md#trade-offs-2

maybe it should go as separated pr

Reply:
We are not generating any classes in runtime, are we?

I'm more interested in the fact it relies on PSR-4 rules, so if there is an issue it would make it visible.

Reply:
Oh, we have some classmap also, not only PSR-4, let's try it then in separate PR and focus here on dropping Travis.

Reply:
> We are not generating any classes in runtime, are we?

one of our deps can, eg polyfill. no clue out of the box